docs/guide/testrunner/configurationfile.md

```diff
@@ -78,7 +78,13 @@ exports.config = {
     // https://docs.saucelabs.com/reference/platforms-configurator
     //
     capabilities: [{
-        browserName: 'chrome'
+        browserName: 'chrome',
+        // to run chrome headless you can use the --headless flag, --disable-gpu is also required for now.
```

Comment:
@tarjei one tiny thing: I think we can remove this line because you already mention the same below